lib/trackler/implementation.rb

```diff
@@ -91,12 +89,19 @@ def readme_title
       problem.name
     end
 
+    def optional_blurb
+      blurb = problem.blurb
+      return '' if problem.description.start_with?(blurb)
+      "#{blurb}\n\n"
+    end
```

Comment:
The blurb will remain present in the metadata file, and can still be accessed by `problem.blurb` as it always has in all the other places that want to use it.

It will just not be prepended to the description when rendering the Readme.